src/components/Meals/MealItem/MealItemForm.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+import Input from "../../UI/Input";
+
+const StyledForm = styled.form`
+   text-align: right;
+   
+   button {
+      font: inherit;
+      cursor: pointer;
+      background-color: #8a2b06;
+      border: 1px solid #8a2b06;
+      color: white;
+      padding: 0.25rem 2rem;
+      border-radius: 20px;
+      font-weight: bold;
+   }
+   
+   button:hover,
+   button:active {
+      background-color: #641e03;
+      border-color: #641e03;
+   }
+`;
+
+const MealItemForm = props => {
+   return (
+      <StyledForm>
+         <Input 
+            label="Amount" 
+            input={{
+               id: 'amount_' + props.id, // this changed!
+               type: 'number',
+               min: '1',
+               max: '5',
+               step: '1',
+               defaultValue: '1',
+           }}
+         />
+         <button type="submit">+ Add</button>
+      </StyledForm>
+   );
+};
```

Comment:
I think you can consider changing the native form submit to sending ajax request. And change the form component to [controlled-component](https://blog.logrocket.com/controlled-vs-uncontrolled-components-in-react/) which the parent keeps the state of the form inputs and sends ajax requests to perform actions to the API server.
We rarely do native form submit in SPA.